rasa/server.py

```diff
@@ -724,9 +698,8 @@ async def status(request: Request) -> HTTPResponse:
 
         return response.json(
             {
-                "model_file": app.agent.path_to_model_archive
-                or app.agent.model_directory,
-                "fingerprint": model.fingerprint_from_path(app.agent.model_directory),
+                "model_file": app.agent.model_path,
+                "fingerprint": app.agent.model_id,  # TODO: is this correct?
```

Comment:
Will this do? Or do we need a model fingerprint?

Reply:
I was getting this mixed up with the fingerprint used for model_server which is different I think. It's a bit confusing though.